assets/js/src/settings/pages/signup_confirmation/email_content.tsx

```diff
@@ -1,4 +1,6 @@
-import React from 'react';
+import React, { Fragment } from 'react';
```

Comment:
Could you please help me understand why do we need `Fragment` here?

Reply:
This is a leftover from the refactoring, I will remove it.